spec/function.dd

```diff
@@ -1408,7 +1417,7 @@ pure void f()
 
 $(H3 $(LNAME2 return-ref-parameters, Return Ref Parameters))
 
-        $(P Note: The `return` attribute is currently only enforced by `dmd`
+        $(P $(B Note): The `return` attribute is currently only enforced by `dmd`
```

Comment:
This could mention the deprecations issued since v2.092 ([which partially enables DIP25 by default](https://dlang.org/changelog/2.092.0.html#dip25)

Reply:
Thanks. I decided to just remove the note, as the deprecation warnings will become errors at some point.